lib/matplotlib/markers.py

```diff
@@ -297,15 +312,33 @@ def set_marker(self, marker):
             self._recache()
 
     def get_path(self):
+        """
+        Return a `.Path` for the primary part of the marker.
+
+        For unfilled markers this is the whole marker, for filled markers,
+        this is the area to be drawn with *markerfacecolor*.
+        """
         return self._path
 
     def get_transform(self):
+        """
+        Return the transform to be applied to the `.Path` from
+        `MarkerStyle.get_path()`.
```

Comment:
Shouldn't pydocstyle be complaining that the first line should only be one line, or do we have that turned off?

Reply:
That would be

> D400 - First line should end with a period

We have currently deactivated that in `.flake8` because we have a couple of docstrings that can't reasonably be cramped into 79 chars.

While it's good to keep the summary sentence short, personally, I don't  see the point in stripping relevant information from the summary only to adhere to that limit. If you have an idea how to shorten the sentence meaningfully to a single line, that'd of course be welcome.

Reply:
Not sure about the transform ones, but the `get_alt_path` one could be written like `get_path`.

Reply:
`get_alt_path`. I have no good idea for wording of the transforms, so I'd leave them pointing to the paths.